### PR TITLE
Updated the club page

### PR DESCRIPTION
**Pull request by Selasi3**

### CODE REVIEW

File: _articles/es/club.md
Comment (kjaymiller) [inline _articles/es/club.md:1]: Delete this file

File: _includes/club.html
Comment (kjaymiller) [inline _includes/club.html:1]: this file should exist at the root and not in includes.

You'll also need to also add the frontmatter at the top

File: _includes/footer.html
Comment (kjaymiller) [inline _includes/footer.html:1]: This is part of a different issue. Please remove this change from the PR

File: _layouts/index.html [hunk 1]
```diff
@@ -13,5 +13,6 @@
   {%- include latest-posts.html -%}
   {%- include join_us.html -%}
   {%- include partnerships.html -%}
+  {%- include club.html -%}
```
Comment (kjaymiller) [inline _layouts/index.html]:
```suggestion
```

File: assets/css/bpd.css
Comment (kjaymiller) [inline assets/css/bpd.css:1]: This is part of a different issue. Please remove from this PR

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Please address the mentioned changes.

**Comment by Selasi3** [comment]
Changes worked on

**Pull request closed without merging**